### ISSUE TITLE
Adding new repository to the model card

### ISSUE THREAD

**Issue by alarmed-ground** [KindaBrazy/LynxHub#13]
**Describe the solution you'd like**
I would like to bring a fork of existing oobabooga local gitlab repository that I had been working on for a while as a model card to LynxHub.

**Additional context**
The fork brings in few bug fixes to superboogav2 and features to add PDF/txt files for RAG and integration in to tensorboard graphs alongside multi node GPU utilisation and other monitoring features.

**Comment by KindaBrazy** [comment]
Is your fork on GitLab? Currently, GitLab isn't supported, but I'll add it in the next version. If it's on GitHub, you can already add it through the module system. Check these links for more details:
[Module Guide](https://github.com/KindaBrazy/LynxHub-Module-Guide)
[WebUI Container](https://github.com/KindaBrazy/LynxHub-Module-Offline-Container)
[Modules List](https://github.com/KindaBrazy/LynxHub-Modules)

**Comment by alarmed-ground** [comment]
Yes, it is on GitLab. Thanks for the guidelines, I'll have a look.

**Comment by KindaBrazy** [comment]
The current version 1.3.1 now supports GitLab, so you can publish your module! If you have any questions or need guidance, feel free to reach out to me on Discord.
